README.md

```diff
@@ -98,10 +98,10 @@ Optimus is currently undergoing heavy development with frequent, breaking API ch
 
 Development of Optimus happens in the open on GitHub, and we are grateful to the community for contributing bugfixes and improvements. Read below to learn how you can take part in improving Optimus.
 
-Read our [contributing guide](docs/contribute/contribution.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Optimus.
+Read our [contributing guide](https://github.com/odpf/optimus/blob/main/docs/docs/contribute/contributing.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Optimus.
```

Comment:
@anuraagbarde Can you update it with this link?
https://odpf.github.io/optimus/contribute/contributing/

Reply:
Done